ang/civicase/DashboardTab.js

```diff
@@ -1,10 +1,109 @@
-(function (angular) {
+(function (angular, $, _) {
   var module = angular.module('civicase');
 
   module.directive('civicaseDashboardTab', function () {
     return {
       restrict: 'E',
+      controller: 'dashboardTabCtrl',
       templateUrl: '~/civicase/DashboardTab.html'
     };
   });
-})(angular);
+
+  module.controller('dashboardTabCtrl', dashboardTabCtrl);
+
+  dashboardTabCtrl.$inject = ['$location', '$scope', 'ContactsDataService', 'formatCase'];
```

Comment:
We are not using $inject statements yet and also not using Square brackets to define/map dependencies, lets follow that.